cpu/mips_pic32mx/Makefile

```diff
@@ -0,0 +1,15 @@
+ifeq ("$(OS)","Windows_NT")
+MIPS_ELF_ROOT?=C:\cross-mips-mti-elf
+else
+MIPS_ELF_ROOT?=/opt/imgtec/Toolchains/mips-mti-elf/2016.05-03
+endif
```

Comment:
They are '?=' ie defaults if nothing else is set and are the default locations the toolchain install too, but can remove them it thats what you prefer.